pages/common/mkvmerge.md

```diff
@@ -14,3 +14,7 @@
 - Extract the subtitle from track 3 of a specific file:
 
 `mkvextract tracks {{path/to/file.mkv}} {{3}}:{{path/to/subs.srt}}`
+
+- Add subtitles to a file:
+
+`mkvmerge -o {{output.mkv}} {{file.mkv}} {{subs.ass}}`
```

Comment:
We prefer long arguments if available.
```suggestion
`mkvmerge --output {{path/to/output.mkv}} {{path/to/file.mkv}} {{path/to/subs.ass}}`
```
Why `subs.ass` and not `.srt`?

Reply:
The subtitle files I needed happened to be `.ass` files. It's okay to change `.ass` to `.srt`.